superset/cli/viz_migrations.py

```diff
@@ -47,66 +87,81 @@ def migrate_viz() -> None:
 @migrate_viz.command()
 @with_appcontext
 @optgroup.group(
-    "Grouped options",
-    cls=RequiredMutuallyExclusiveOptionGroup,
+    cls=RequiredAnyOptionGroup,
 )
 @optgroup.option(
     "--viz_type",
     "-t",
-    help=f"The viz type to migrate: {', '.join(list(VizType))}",
+    help=f"The viz type to upgrade: {', '.join(list(VizType))}",
+    type=str,
 )
-def upgrade(viz_type: str) -> None:
+@optgroup.option(
+    "-ids",
```

Comment:
Shouldn't this have two dashes rather than one? Also why not use the multiple options `--id 1 --id 2 ...` so that way you don't need to split a string and the types will be correct, i.e., the following will correctly fail during the validation phase:

```
... --id 1 --id foo 
```

Reply:
Change it to use double dashes 👍🏼

Reply:
Change it to use a multi-value parameter `--id`